src/Files.App/Commands/CommandCodes.cs

```diff
@@ -16,5 +16,9 @@ public enum CommandCodes
 		EmptyRecycleBin,
 		RestoreRecycleBin,
 		RestoreAllRecycleBin,
+
+		// favorites
```

Comment:
Should it be "Favorites"?
```suggestion
		// Favorites
```